ssl/t1_trce.c

```diff
@@ -423,7 +423,7 @@ static ssl_trace_tbl ssl_ciphers_tbl[] = {
     {0xCCAC, "TLS_ECDHE_PSK_WITH_CHACHA20_POLY1305"},
     {0xCCAD, "TLS_DHE_PSK_WITH_CHACHA20_POLY1305"},
     {0xCCAE, "TLS_RSA_PSK_WITH_CHACHA20_POLY1305"},
-    {0x0D01, "TLS_AES_128_GCM_SHA256"},
+    {0x1301, "TLS_AES_128_GCM_SHA256"},
```

Comment:
which leads to the obvious question as to why we're not using the TLS...CK... define's masked off?
and the semi-obvious next question, why we're not using the TXT defines (which I hate and want to get rid of)